package.json

```
@@ -4,6 +4,7 @@
     "limax": "^2.1.0"
   },
   "dependencies": {
-    "@joshdb/core": "^1.1.3"
+    "@joshdb/core": "^1.1.3",
+    "file-system": "^2.2.2"
   }

```

```suggestion
    "@joshdb/core": "^1.1.3"
  },
  "devDependencies": {
    "file-system": "^2.2.2"
  }
```
This should be a dev-dep due to it being used for tests. Also, make sure to regen the lockfile after making this change.